packages/odyssey-storybook/src/components/odyssey-mui/Link/Link.stories.tsx

```diff
@@ -68,7 +67,7 @@ export const WithIcon = Template.bind({});
 WithIcon.args = {
   href: "#anchor",
   children: "Info link",
-  icon: <InformationCircleFilledIcon />,
+  icon: <Icon name="information-circle-filled" />,
```

Comment:
We shouldn't change these and should instead use our exported icon components from MUI.